client/src/Components/Button/StyledImportProspectButtonOutline.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import { Button, withStyles } from "@material-ui/core";
+
+const styles = {
+  root: {
+    backgroundColor: "#F4F6FC",
+    border: "1px solid #2AA897",
+    borderRadius: 7,
+    color: "#000000",
+    fontWeight: "bold",
+    fontSize: ".8rem",
+    height: 54,
+    width: 140
+  }
+};
+
+const StyledImportProspectButtonOutline = props => {
```

Comment:
I liked how you created a specific styled button - I feel like this name though is really specific though, so it's not that reusable - I'd generalize the name here